CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Vortex
+
+Welcome! Thank you for your interest in Vortex. Before contributing, please
+review the guidelines below.
+
+## Reporting Issues
+
+Found a bug? Have an improvement to suggest? Please file a
+[GitHub issue](https://github.com/spiraldb/vortex/issues).
+Before you create a new issue, please ensure that a relevant issue doesn't
+already exist by running a quick search of existing issues.
+If you're unable to find an open issue, then please open a new one.
+
+## Code Contributions
+
+The contribution process is outlined below:
+
+1. Start a discussion by creating or commenting on a GitHub Issue (unless it's a very minor change).
+
+2. Implement the change.
+    * If the change is large, consider posting a draft pull request (PR)
+      with the title prefixed with [WIP], and share with the team to get early feedback.
+    * Give the PR a clear, brief description; this will be the commit message
+      when the PR is merged.
+    * Make sure the PR passes all CI tests.
+
+3. Open a PR to indicate that the change is ready for review.
+    * Sign the [Contributor License Agreement](https://gist.github.com/lwwmanning/e246ddd906c18ffb79bfef61a888b080) (CLA) when prompted. 
```

Comment:
is this correct?

Reply:
At least move it to the spiraldb org

Reply:
Orgs don't support gists, and cla-assistant requires a gist. 

https://github.com/cla-assistant/cla-assistant/issues/926

Reply:
I'll just change it to https://cla.spiraldb.com and we can manage via Cloudflare